lib/Reader/readerir.cpp

```diff
@@ -1678,6 +1762,44 @@ Type *GenIR::getClassTypeWorker(
     }
   }
 
+  uint32_t EEClassSize = 0;
+  bool HaveClassSize = false;
+  if (!IsRefClass) {
+    try {
+      EEClassSize = getClassSize(ClassHandle);
+      HaveClassSize = true;
+    } catch (...) {
```

Comment:
No, there isn't. I asked @jkotas about whether such a way could be added and his response was
"I think it would be better if the JIT-EE interface method had an argument that would say whether the layout of struct should be recorded as method pre-requisite."
He later clarified that the way it would work was "You can pass “do not record the size as pre-requisite” when you are mirroring the type system, and then call the API again with “do record size as pre-requisite” as you actually used the information later.

Or you can always pass in “do not record the size as pre-requisite” - assuming that we will eventually get framework build-time tooling in place that marks valuetype layout changes as breaking changes."

I thought it wasn't straightforward for us to do what he suggested and just decided to go with catching the exception for now. I may revisit this later depending on how soon that build tooling is expected to be in place. (Jan did say it was the plan of record).